internal/pkg/mount/mount.go

```diff
@@ -67,6 +68,14 @@ func mountMountpoint(mountpoint *Point) (skipMount bool, err error) {
 
 	if !skipMount {
 		if err = mountpoint.Mount(); err != nil {
+			if errors.Is(err, unix.ENODEV) {
```

Comment:
this should probably use the new option?

Reply:
yeah...